client/components/environment-badge/index.jsx

```diff
@@ -10,6 +10,12 @@ export function ReactQueryDevtoolsHelper() {
 	/* eslint-enable wpcalypso/jsx-classname-namespace */
 }
 
+export function AccountSettingsHelper() {
+	/* eslint-disable wpcalypso/jsx-classname-namespace */
+	return <div className="environment is-account-settings" />;
+	/* eslint-enable wpcalypso/jsx-classname-namespace */
+}
```

Comment:
What about calling the helper just `<LangHelper>`? It changes only language and is unlikely to do anything else.

The `eslint-disable` comments are no longer needed these days. The `wpcalypso/jsx-classname-namespace` rule has changed a few months ago and now accepts more classnames.

Reply:
> What about calling the helper just `<LangHelper>`? It changes only language and is unlikely to do anything else.

@jsnajdr To create space for it to also switch the `Dashboard Color Scheme`.

> The `eslint-disable` comments are no longer needed these days. The `wpcalypso/jsx-classname-namespace` rule has changed a few months ago and now accepts more classnames.

Ok, removed them in https://github.com/Automattic/wp-calypso/pull/69073

Reply:
> To create space for it to also switch the Dashboard Color Scheme.

Oh yes, that's a neat idea 🙂